www/front_src/src/PollerWizard/pollerStep1/index.tsx

```diff
@@ -114,6 +114,9 @@ const FormPollerStepOne = ({
 
   const waitListOption = waitList?.map((c) => ({ id: c.ip, name: c.ip }));
 
+  const hasError = (stateName): string | undefined =>
```

Comment:
The function name doesn't sound correct, as it isn't a check for the error existence, but rather a retrieve of it's value. getError would sound more accurate to me